<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>979</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>주가 분석 프로그램 (Prod. 인생력전)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>831</width>
      <height>901</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="main_tab">
     <attribute name="title">
      <string>데이터 분석</string>
     </attribute>
     <widget class="QLineEdit" name="code_lineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="codelabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>종목코드  : </string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_lineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="namelabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>종목명     : </string>
      </property>
     </widget>
     <widget class="QLabel" name="startdate">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>분석일자  : </string>
      </property>
     </widget>
     <widget class="QLabel" name="enddate">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>~            :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="start_dateEdit">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>31</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="end_dateEdit">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>80</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="mon_up_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>570</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 상승</string>
      </property>
     </widget>
     <widget class="QLabel" name="mon_down_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>600</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 하락</string>
      </property>
     </widget>
     <widget class="QLabel" name="mon_eq_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>630</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 동일</string>
      </property>
     </widget>
     <widget class="QLabel" name="tue_eq_label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>630</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 동일</string>
      </property>
     </widget>
     <widget class="QLabel" name="tue_down_label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>600</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 하락</string>
      </property>
     </widget>
     <widget class="QLabel" name="tue_up_label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>570</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 상승</string>
      </property>
     </widget>
     <widget class="QLabel" name="wed_up_label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>570</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 상승</string>
      </property>
     </widget>
     <widget class="QLabel" name="wed_eq_label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>630</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 동일</string>
      </property>
     </widget>
     <widget class="QLabel" name="wed_down_label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>600</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 하락</string>
      </property>
     </widget>
     <widget class="QLabel" name="thu_down_label">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>600</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 하락</string>
      </property>
     </widget>
     <widget class="QLabel" name="thu_up_label">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>570</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 상승</string>
      </property>
     </widget>
     <widget class="QLabel" name="thu_eq_label">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>630</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 동일</string>
      </property>
     </widget>
     <widget class="QLabel" name="fri_eq_label">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>630</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 동일</string>
      </property>
     </widget>
     <widget class="QLabel" name="fri_up_label">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>570</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 상승</string>
      </property>
     </widget>
     <widget class="QLabel" name="fri_down_label">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>600</y>
        <width>56</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>% 하락</string>
      </property>
     </widget>
     <widget class="PlotWidget" name="graphWidget" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>731</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchbtn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>501</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>분석</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="anal_start_val_edit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>690</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="anal_start_value">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>690</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>분석 시작일 종가    :</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="anal_end_val_edit">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>690</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="anal_end_value">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>690</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>분석 시작일 종가    :</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="diff_data_edit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>740</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="diff_data_label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>740</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>시작일 대비           :</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="rcmd_buy_date_edit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>810</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="rcmd_buy_date_label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>810</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>매수 추천 요일       :</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="rcmd_sell_date_label">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>810</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>매도 추천 요일       :</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="rcmd_sell_date_edit">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>810</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="list_tab">
     <attribute name="title">
      <string>종목 LIST</string>
     </attribute>
     <widget class="QTableWidget" name="listCorp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>351</width>
        <height>801</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>251</red>
            <green>251</green>
            <blue>251</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>251</red>
            <green>251</green>
            <blue>251</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>31</number>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>종목코드</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>기업명</string>
       </property>
      </column>
      <column/>
      <column/>
     </widget>
     <widget class="QPushButton" name="btnSearch">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>검색</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtBoxName">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comBoxProp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>종목코드</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>기업명</string>
       </property>
      </item>
     </widget>
     <widget class="QGroupBox" name="groupInfoCorp">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>441</width>
        <height>841</height>
       </rect>
      </property>
      <property name="title">
       <string>기업 정보</string>
      </property>
      <widget class="QTableWidget" name="detailCorp">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>421</width>
         <height>601</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectColumns</enum>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>421</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="spacing">
         <number>12</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lblCorp">
          <property name="font">
           <font>
            <family>맑은 고딕</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Corperation</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QProgressBar" name="pgBarSearch">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>820</y>
         <width>421</width>
         <height>10</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
